### ISSUE TITLE
Better error message when trying to load a tga image

### ISSUE THREAD

**Issue by ombre5733** [godotengine/godot#8106]
I tried to import a COLLADA file referencing a texture in TGA format. Godot brings up an error saying that the image could not be found. I first thought that something is wrong with the path. However, as soon as I changed the format to PNG everything worked.

So I suggest to re-write the error message to something like "Image format not supported".

**Comment by supagu** [comment]
I believe tga support is coming:
https://github.com/godotengine/godot/issues/7686

**Comment by fahall** [comment]
I'll take a look at this. This is my first attempt at contributing to development, so we'll see =).

I imagine this 'Image format not supported' message should be used for other non-supported formats as well, not just .TGA. Someone please let me know if that is not the case.

**Comment by reduz** [comment]
TGA is now supported, so please try again :P

**Comment by vnen** [comment]
In any case, are there other image formats that Collada supports but Godot does not? If that's the case, a proper error message would still be good.

**Comment by fahall** [comment]
My thought is that even if Godot currently supports all of the Collada formats, it is possible that in the future they will be out of sync again. If that happens it'd be nice to have a proper error message provided.

**Comment by fahall** [comment]
I cannot recreate this error. The current version of Godot appears to support TGA files. I made a bad file called `bad_image.imagetype` to test what happens when an unrecognized image filetype is imported. I got this error: `Error loading file: Not a resource!`. This message seems clear to me. Another option might be something along the lines of `Error loading file: Not a supported resource type!`, but I imagine that should be submitted as a different issue.

It is possible that a file supported by Collada and not supported by Godot may provide a different error message; however, I have not been able to recreate the error. 

Unless someone can provide instructions as to how to recreate this error, I recommend closing this issue.
